core/strings/conversion.odin

```diff
@@ -4,6 +4,20 @@ import "core:io"
 import "core:unicode"
 import "core:unicode/utf8"
 
+/*
+Converts invalid UTF-8 sequences in the input string `s` to the `replacement` string.
+
+*Allocates Using Provided Allocator*
+
+Inputs:
+- s: Input string that may contain invalid UTF-8 sequences.
+- replacement: String to replace invalid UTF-8 sequences with.
+- allocator: (default: context.allocator).
```

Comment:
This will be formatted incorrectly on the website.
You need to have 2 trailing spaces in markdown to add a break and not have everything be on a single line.
i.e.
`Inputs:  ` rather than `Inputs:`

Furthermore to make this stand out more on the website it might be beneficial to make the `Inputs`, `Returns`, etc. to be bold.
i.e.
`**Inputs**  `

Note you don't need to do this for `Example:` since that has its own formatting rules when generating the website documentation.